src/views/poi_panel.dot

```diff
@@ -119,7 +119,7 @@
             {{~ this.categories:category:index }}
               <button class="service_panel__category" type="button" {{= click(this.openCategory, this, category) }}>
                 <div class="service_panel__category__icon" style="background: {{= category.backgroundColor }}">
-                  <span class="{{? index < 9 || index == 10 }}icon{{??}}category__new-icon{{?}} icon-{{= category.iconName }}"></span>
```

Comment:
`category__new-icon` can also be removed from categories.scss

Reply:
sure!